### PR TITLE
LPS-178051 Adapt SEO, Open Graph and Meta Tags views

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/open_graph.jsp [hunk 1]
```diff
@@ -103,21 +103,35 @@ Layout selLayout = layoutsSEODisplayContext.getSelLayout();
...
-						<aui:input label="<%= StringPool.BLANK %>" name="openGraphImageTitle" placeholder="image" readonly="<%= true %>" title="image" type="text" value="<%= layoutsSEODisplayContext.getOpenGraphImageTitle() %>" wrapperCssClass="mb-3" />
-
-						<aui:button-row cssClass="mt-0">
-							<aui:button aria-label='<%= LanguageUtil.format(locale, "select-x", "image") %>' name="openGraphImageButton" value="select" />
-							<aui:button aria-label='<%= LanguageUtil.format(locale, "clear-x", "image") %>' name="openGraphClearImageButton" value="clear" />
-						</aui:button-row>
+						<div class="align-items-center d-flex">
+							<aui:input label="<%= StringPool.BLANK %>" name="openGraphImageTitle" placeholder="no-image-is-loaded" readonly="<%= true %>" title="image" type="text" value="<%= layoutsSEODisplayContext.getOpenGraphImageTitle() %>" wrapperCssClass="flex-grow-1 mb-0 mr-2" />
+
+							<aui:button-row cssClass="mt-0">
```
Comment (victorg1991) [inline modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/open_graph.jsp:116]: I believe we can remove this aui tag and just use a plain div 🔥

File: modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/open_graph.jsp [hunk 2]
```diff
@@ -103,21 +103,35 @@ Layout selLayout = layoutsSEODisplayContext.getSelLayout();
...
 						<label class="control-label"><liferay-ui:message key="image" /> <liferay-ui:icon-help message="open-graph-image-help" /></label>
 
-						<aui:input label="<%= StringPool.BLANK %>" name="openGraphImageTitle" placeholder="image" readonly="<%= true %>" title="image" type="text" value="<%= layoutsSEODisplayContext.getOpenGraphImageTitle() %>" wrapperCssClass="mb-3" />
-
-						<aui:button-row cssClass="mt-0">
-							<aui:button aria-label='<%= LanguageUtil.format(locale, "select-x", "image") %>' name="openGraphImageButton" value="select" />
-							<aui:button aria-label='<%= LanguageUtil.format(locale, "clear-x", "image") %>' name="openGraphClearImageButton" value="clear" />
-						</aui:button-row>
+						<div class="align-items-center d-flex">
+							<aui:input label="<%= StringPool.BLANK %>" name="openGraphImageTitle" placeholder="no-image-is-loaded" readonly="<%= true %>" title="image" type="text" value="<%= layoutsSEODisplayContext.getOpenGraphImageTitle() %>" wrapperCssClass="flex-grow-1 mb-0 mr-2" />
```
Comment (victorg1991) [inline modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/open_graph.jsp:114]: Can we add the click on the input too? so the selector can be opened too

**Comment by victorg1991** [comment]
Divino @sandrodw3  two minor comments :P  take a look at them and let me know we can pass this PR like it is too

File: modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/open_graph.jsp (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-seo-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/open_graph.jsp:116]: > Divino @sandrodw3 two minor comments :P take a look at them and let me know we can pass this PR like it is too

Thanks for the suggestions @victorg1991, I'll apply both 😄

**Comment by sandrodw3** [comment]
> Divino @sandrodw3 two minor comments :P take a look at them and let me know we can pass this PR like it is too

Thanks for the suggestions @victorg1991, I'll apply both smile

**Comment by sandrodw3** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by alessandromalves** [comment]
The reported failures will only require a fix at the tests

**Pull request closed without merging**